src/proxies/clientSDKCacheProxy.ts

```diff
@@ -0,0 +1,36 @@
+import { CogniteClient } from '@cognite/sdk';
+import { ClientSDKCache } from '../cache/ClientSDKCache';
+import { ClientApiKeys, ClientApiTypes } from './types';
+
+type CachedKeys = keyof ClientSDKCache;
+type CachedTypes = ClientSDKCache[CachedKeys];
+
+export const wrapInCacheProxy = (
+  client: CogniteClient,
+  cache: ClientSDKCache
+): CogniteClient => {
+  const clientHandler: ProxyHandler<CogniteClient> = {
+    get(target, name: ClientApiKeys | CachedKeys) {
+      if (cache[name as keyof ClientSDKCache]) {
```

Comment:
don't we need to exclude here same way we do in logProxy?

Reply:
no, cause `ClientSDKCache` has a limited number of keys that we provided and what we expect to handle. Right now, it supports only `'assets'` and `'timeseries'`

Reply:
ok!